app.py

```diff
@@ -0,0 +1,15 @@
+from st_pages import Page, Section, show_pages, add_page_title
+from frontend.form import create_form
```

Comment:
I think I know what problem you are referring to regarding relative imports and how to solve it.

We can look into it during our session, but we may as well first focus on other topics